### DISCUSSION TITLE
how to correct this error while editing fluentd config file.

### DISCUSSION THREAD

**Discussion by swapnil1518** [fluent/fluentd#3934] [Q&A]
error converting YAML to JSON: yaml: line 457:
found a tab character where an indentation space is expected.

**Comment by fujimotos** [comment]
> found a tab character where an indentation space is expected.

YAML does not accept tabs as indentation. That error message
basically says that you have some tabs in your configuration file. 

Find tab characters and replace them with spaces using your editor.
